NOLF2/ClientShellDLL/ClientShellShared/HUDChatInput.cpp

```diff
@@ -29,6 +29,7 @@ CHUDChatInput::CHUDChatInput()
 	m_pStr = LTNULL;
 	m_eLevel = kHUDRenderText;
 	m_bTeamMessage = false;
+	m_szChatStr[0] = 0;
```

Comment:
although correct, I'd prefer `'\0'` since it is a char array

Reply:
Done.

Reply:
You even do a test later on in rezmgr between `szPath[0]` and `'\0'`
thanks for keeping it consistent.